zuul-integration-test/src/test/java/com/netflix/zuul/integration/IntegrationTest.java

```diff
@@ -117,6 +128,8 @@ static void beforeAll() {
         int wireMockPort = wireMockExtension.getPort();
         AbstractConfiguration config = ConfigurationManager.getConfigInstance();
         config.setProperty("zuul.server.netty.socket.force_nio", "true");
+        // run with a single event loop to make tests more deterministic, especially for connection pooling
+        config.setProperty("zuul.server.netty.threads.worker", "1");
```

Comment:
Should we also force at-least one test to run with 1 vs 2 workers?

Reply:
yeah, that's a good call